docs/standard/components.md

```diff
@@ -54,27 +53,34 @@ Mono is the runtime used by Xamarin apps.  It implements the .NET Standard, whic
 
 To learn more about Mono, see the [Mono documentation](http://www.mono-project.com/docs/).
 
+## .NET runtimes
+
+A runtime is the execution environment for a managed program. The OS is part of the runtime environment but is not part of the .NET runtime. Here are some examples of .NET runtimes:
+ 
+ - Common Language Runtime (CLR) for .NET Framework
+ - Core Common Language Runtime (CoreCLR) for .NET Core
+ - .NET Native for Universal Windows Platform 
+
 ## .NET tooling and common infrastructure
 
-Tooling for .NET is also common across each implementation of .NET.  These include, but are not limited to:
+Some tooling for .NET is common across all implementations of .NET.  These include, but are not limited to:
 
-* The .NET languages and their compilers
-* Runtime components, such as the JIT and Garbage Collector
-* .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
-* MSBuild, the build engine used to build projects
-* NuGet, Microsoft's package manager for .NET
-* The .NET Core CLI, a cross-platform command-line interface for building .NET projects
-* Open Source build orchestration tools, such as CAKE and FAKE
+- The .NET languages and their compilers
+- .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
```

Comment:
Commas inside of quotes, but I admit doing something different in cases like this because these are file extensions (types), too, and there are similar cases where I didn't want there to be any confusion about punctuation marks and technical terms. I've made good use of emphasis ... just a suggestion ...

> .NET project system (sometimes known as \*csproj\*, \*vbproj\*, or \*fsproj\*)

.NET project system (sometimes known as *csproj*, *vbproj*, or *fsproj*)